<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_strFilenameProps</class>
 <widget class="QDialog" name="Dialog_strFilenameProps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Content</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>218</red>
               <green>218</green>
               <blue>108</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>218</red>
               <green>218</green>
               <blue>108</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>218</red>
               <green>218</green>
               <blue>108</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>218</red>
               <green>218</green>
               <blue>108</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="2">
           <widget class="QPushButton" name="pushButton_file">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>283</width>
              <height>49</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>283</width>
              <height>49</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_description">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Selected Filename</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_value">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Properties</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" rowspan="2">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Properties</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_numberElements">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Format</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Decoder Hint Type</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="lineEdit_decoder_hint">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_format">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Number Elements</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Category</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="lineEdit_category"/>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_propertyid">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Id</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1" rowspan="2">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Allowed State</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QRadioButton" name="radioButton_state_init">
               <property name="text">
                <string>init</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_state_selected">
               <property name="text">
                <string>selected</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_state_active">
               <property name="text">
                <string>active</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_state_prepared">
               <property name="text">
                <string>prepared</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_state_processing">
               <property name="text">
                <string>processing</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Allowed Thread</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QRadioButton" name="radioButton_thread_init">
               <property name="text">
                <string>init</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_thread_async">
               <property name="text">
                <string>async</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_thread_sync">
               <property name="text">
                <string>sync</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Access</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QRadioButton" name="radioButton_ronly">
               <property name="text">
                <string>read only</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_rwcontent">
               <property name="text">
                <string>rw content</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_rwfull">
               <property name="text">
                <string>rw full</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_strFilenameProps</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_strFilenameProps</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_value</sender>
   <signal>editingFinished()</signal>
   <receiver>Dialog_strFilenameProps</receiver>
   <slot>textEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_state_init</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_strFilenameProps</receiver>
   <slot>anyRadioButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_state_selected</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_strFilenameProps</receiver>
   <slot>anyRadioButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>-13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_state_active</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_strFilenameProps</receiver>
   <slot>anyRadioButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>-24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_state_prepared</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_strFilenameProps</receiver>
   <slot>anyRadioButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>-17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_state_processing</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_strFilenameProps</receiver>
   <slot>anyRadioButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>-7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_thread_init</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_strFilenameProps</receiver>
   <slot>anyRadioButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_thread_async</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_strFilenameProps</receiver>
   <slot>anyRadioButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_thread_sync</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_strFilenameProps</receiver>
   <slot>anyRadioButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_ronly</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_strFilenameProps</receiver>
   <slot>anyRadioButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_file</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_strFilenameProps</receiver>
   <slot>selectFilename()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>textEdited()</slot>
  <slot>comboBox_decoder_activated(int)</slot>
  <slot>anyRadioButtonClicked()</slot>
  <slot>selectFilename()</slot>
 </slots>
</ui>
